src/moscot/problems/space/_mixins.py

```diff
@@ -414,10 +415,13 @@ def correlate(  # type: ignore[misc]
             - ``'pearson'`` - `Pearson correlation <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`_.
             - ``'spearman'`` - `Spearman's rank correlation
                 <https://en.wikipedia.org/wiki/Spearman%27s_rank_correlation_coefficient>`_.
-        groupby
-            Optional `obs` field in `AnnData` to compute correlations over categorical groups.
         device
             Device where to transfer the solutions, see :meth:`~moscot.base.output.BaseSolverOutput.to`.
+        groupby
+            Optional `obs` field in `AnnData` to compute correlations over categorical groups.
```

Comment:
I know this was there before but can we do ` `:attr:`~anndata.AnnData.obs ` `

Reply:
:attr:`~anndata.AnnData.obs

Reply:
great, done!